lib/rustler_precompiled.ex

```diff
@@ -51,23 +76,30 @@ defmodule RustlerPrecompiled do
           end
 
         {:error, precomp_error} ->
-          error = "Error while downloading precompiled NIF: #{precomp_error}\n\n"
+          raise """
+          Error while downloading precompiled NIF: #{precomp_error}.
 
-          if Mix.env() == :prod do
-            raise error
-          else
-            Logger.debug(error)
-          end
+          You can force the project to build from scratch with:
+
+              config :rustler_precompiled, :force_build, #{otp_app}: true
+          """
```

Comment:
We should move the whole building of the error message to `__using__/2`. In here we just raise or call `Logger.debug`.